common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -231,21 +231,22 @@
          * not differs from current speed.
          */
         setSpeed: function(speed, silent, forceUpdate) {
-            if (speed !== this.currentSpeed || forceUpdate) {
+            var newSpeed = this.state.speedToString(speed);
+            if (newSpeed !== this.currentSpeed || forceUpdate) {
                 this.speedsContainer
                     .find('li')
-                    .siblings("li[data-speed='" + speed + "']");
+                    .siblings("li[data-speed='" + newSpeed + "']");
 
-                this.speedButton.find('.value').text(speed + 'x');
-                this.currentSpeed = speed;
+                this.speedButton.find('.value').text(newSpeed + 'x');
+                this.currentSpeed = newSpeed;
 
                 if (!silent) {
-                    this.el.trigger('speedchange', [speed, this.state.speed]);
+                    this.el.trigger('speedchange', [newSpeed, this.state.speed]);
```

Comment:
<img width="697" alt="screen shot 2019-03-04 at 18 24 19" src="https://user-images.githubusercontent.com/4252738/53736164-c317f380-3eaa-11e9-91bd-2b83e3e50bcf.png">
I think it makes sense to make both speeds consistent.

Reply:
That's the [place](https://github.com/edx/edx-platform/blob/master/common/lib/xmodule/xmodule/js/src/video/09_events_plugin.js#L101-L107) where this information is being logged. I will add the snippet to convert these speeds so that logs don't seem confusing.

Reply:
I only found this trigger sending to different data types of speeds. Will be good,  if you are going to make them consistent.